010_experiments/010_experiments.tex

```diff
@@ -1,8 +1,382 @@
+
+The average age of agents, denoted by \Cref{fig:Average-Age-as-Shuffle-Period-Decreases}, can be see to follow the inverse trend to death rates which is as expected. This trend being, that the longer the reshuffle period the lower the average age of an agent at death. 
+\begin{figure}[H] %Average Age As Shuffle Period Decreases
+    \centering
+    \begin{minipage}{0.8\textwidth}
+        \begin{tikzpicture}
+            \begin{axis}[
+                width=\textwidth,
+                axis y line*=left,
+                ymin=0,
+                xmin=0,
+                xlabel=Shuffle Period (no. of days),
+                ylabel=Average Age of All Agents,
+                ymajorgrids=true,
+                xmajorgrids=true,
+                grid style=dashed,
+                legend pos=south east
+            ]
+            \addplot[mark=*,blue]
+                coordinates{
+                    (14, 24.798)
+                    (7, 30.327)
+                    (2, 33.138)
+                }; \label{leg:reshuffle-ticks-per-floor1}
+            \addplot[mark=*,red]
+                coordinates{
+                    (14, 28.087)
+                    (7, 28.096)
+                    (2, 32.695)
+                }; \label{leg:reshuffle-ticks-per-floor10}
+            \addplot[mark=*,green]
+                coordinates{
+                    (14, 24.034)
+                    (7, 27.36)
+                }; \label{leg:reshuffle-ticks-per-floor5}
+            \addlegendimage{/pgfplots/refstyle=leg:reshuffle-ticks-per-floor1}
+            \addlegendentry{1 Tick per Floor}
+            \addlegendimage{/pgfplots/refstyle=leg:reshuffle-ticks-per-floor5}
+            \addlegendentry{5 Ticks per Floor}
+            \addlegendimage{/pgfplots/refstyle=leg:reshuffle-ticks-per-floor10}\addlegendentry{10 Ticks per Floor}
+            \end{axis}
+        \end{tikzpicture}
+    \end{minipage}
+    \caption{Average Age in the Tower as Reshuffle Period Increases}
+    \label{fig:Average-Age-as-Shuffle-Period-Decreases}
+\end{figure}
+
+\Cref{fig:Total-Accepted-Treaties-As-Shuffle-Period-Decreases} and \Cref{fig:Total-Rejected-Treaties-As-Shuffle-Period-Decreases} show that significant numbers of treaties are being exchanged at all reshuffle periods, demonstrating that a social network is being formed. However, as the reshuffle period decreases the number of treaties accepted and rejected and so the total number of treaties proposed decreases. This is as expected since, if agents have the same neighbours for longer periods of time, they need to exchange fewer new treaties, given that each proposed treaty will be agreed upon by neighbours for longer and so neighbours will not need to send each other new treaties. 
```

Comment:
'However, as the reshuffle period decreases the number of treaties accepted and rejected and so the total number of treaties proposed decreases.' needs rewording

Reply:
```suggestion
\Cref{fig:Total-Accepted-Treaties-As-Shuffle-Period-Decreases} and \Cref{fig:Total-Rejected-Treaties-As-Shuffle-Period-Decreases} show that significant numbers of treaties are being exchanged at all reshuffle periods, demonstrating that a social network is being formed. However, as the reshuffle period decreases the number of treaties accepted and rejected, the total number of treaties proposed also decreases. This is as expected since, if agents have the same neighbours for longer periods of time, they need to exchange fewer new treaties, given that each proposed treaty will be agreed upon by neighbours for longer and so neighbours will not need to send each other new treaties. 
```